Chrysalis-Papilio/Soen490-Papilio-Website
# PAPILIO-86 Website home page created

## General Information


**Issue number**: Papilio-86


**Task description:**
Home page for the website was added.

## Manual Testing
To test the validation follow these steps:
1. Start the website
`npm start`
2. Home page should be visible as soon as the website opens


## Testing 
To run all unit tests, run the following code: `npm test`
To run the coverage, run the following code: `npm run coverage`


## Notes
- Need to add 'about' sections to the second part of the home page.
- Need to add question submission section.
- Need to add footer
